files/filebrowser/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "FileBrowser",
+  "repo": "jonataswalker/FileBrowser",
+  "files": {
+    "basePath": "build/",
+    "include": ["filebrowser.js", "filebrowser.css", "filebrowser.min.css"]
```

Comment:
Why do you have both filebrowser.min.js and filebrowser.js in the repo? The both look to be minified